ui/desktop/app/templates/scopes/scope/projects/targets.hbs

```diff
@@ -1,3 +1,63 @@
 {{page-title (t 'resources.target.title_plural')}}
 
+{{#each @model as |target|}}
+  {{#each target.sessions as |session|}}
+    {{#if session.started_desktop_client}}
+      {{#unless session.acknowledged}}
+        <DialogDetail
+          @heading={{concat 'Target connection details: ' session.id}}
+          @dismiss={{route-action 'acknowledge' session}}
+          @dismissButtonText={{t 'actions.ok'}}
+          as |dialogDetail|
+        >
+          <dialogDetail.notification
+            @style='success'
+            @heading='Successfully connected'
```

Comment:
Use translation helper for heading string to future proof multi-language support, something like `{{t resources.target.messages.connection-success}}`

Reply:
Actually, can you double check this file for naked string? There are a few within heading tags. Ty!

Reply:
Absolutely! I missed the translation helper, but now you mention I will check the file 😉 .

Thanks for the heads up with the translation helper, I will put it in the top of my mind while working in other files.

Reply:
I checked this file, now it uses the translation helper.